File: data/c#/pattern_14/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        int x;
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = 1; i <= num; i++)
        {
            x = num - i + 1;
            for (int j = 1; j <= num; j++)
            {
                Console.Write("{0,2:D }", x);
                x = x + num;
            }
            Console.Write("\n");
        }
    }
}

File: data/c++/pattern_14/code.cpp
#include <iostream>
using namespace std;

int main(void)
{
    int num;
    int x;
    cout << "Enter the number of rows and columns for the square: ";
    cin >> num;

    for (int i = 1; i <= num; i++)
    {
        x = num - i + 1;
        for (int j = 1; j <= num; j++)
        {
            cout << x << " ";
            x = x + num;
        }
        cout << endl;
    }

    return 0;
}
